features/native_session_tracking.feature

```diff
@@ -2,6 +2,8 @@ Feature: NDK Session Tracking
 
 Scenario: Stopped session is not in payload of unhandled NDK error
     When I run "CXXStopSessionScenario"
+    And I wait a bit
```

Comment:
What does this do, and why do we required two of them?  Is it possible to detect if an app is running through CLI somehow?

Reply:
This step will wait for a set amount of time that is lower locally, but increased on CI. I believe it requires 2 as that's what was used in previous scenarios involving sessions and native errors.
https://github.com/bugsnag/bugsnag-android/blob/31d75900a89f1de6f67047fb98ba80eead2a4943/features/steps/build_steps.rb#L1

I believe it might be possible to detect if an app is running through adb but we don't currently have that setup.

Reply:
Is it relied on somewhere else as well, or could we just bump the time up in a single step?

Reply:
I believe the timing is relied on in other places.